frontend/src/components/modules/Game/OfficialGameInterface.tsx

```diff
@@ -0,0 +1,197 @@
+import React, { useState, useEffect } from "react";
+import {
+  Table,
+  Typography,
+  TableBody,
+  TableRow,
+  TableCell,
+  Button,
+  Dialog,
+  DialogContent,
+  DialogTitle,
+  Radio,
+  RadioGroup,
+  FormControlLabel
+} from "@mui/material";
+import "../../common/Style/common.css";
+
+interface Cells {
```

Comment:
Instead of writing out the rows/columns like so, couldn't we use a 2 dimensional array?  Would definitely make this component a lot simpler since we wouldn't have to explicitly define everything like so. The div elements could also be mapped in a more straightforward manner.